cmake/toolchains/ARM.cmake

```diff
@@ -1,14 +1,16 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-set(CMAKE_ASM_COMPILER    "armasm")
-set(CMAKE_C_COMPILER      "armclang")
-set(CMAKE_CXX_COMPILER    "armclang")
-set(CMAKE_AR              "armar")
-set(ELF2BIN               "fromelf")
+set(CMAKE_ASM_COMPILER "armasm")
+set(CMAKE_C_COMPILER "armclang")
+set(CMAKE_CXX_COMPILER "armclang")
+set(CMAKE_AR "armar")
+set(ARM_ELF2BIN "fromelf")
+set_property(GLOBAL PROPERTY ELF2BIN ${ARM_ELF2BIN})
 
+get_property(mbed_studio_arm_compiler GLOBAL PROPERTY MBED_STUDIO_ARM_COMPILER)
```

Comment:
same as in the toolchain (global vs mbed-os)

Reply:
Please see response above as it is the same property.